components/shared/modals/ModalBody.vue

```diff
@@ -68,33 +74,37 @@ const props = withDefaults(
     contentClass?: string
     scrollable?: boolean
     customSkeletonTitle?: string
+    estimatedTime?: number
   }>(),
   {
     modalWidth: '25rem',
     modalMaxWidth: '30rem',
     modalMaxHeight: '70vh',
     scrollable: true,
     contentClass: 'pt-4 pb-5 px-6',
+    estimatedTime: undefined,
```

Comment:
undefined "default" value is pointless

Reply:
> undefined "default" value is pointless
yep is the default anyways 

but didn't we have a lint rule that forced default values

fixed